### PR TITLE
Add CreateBudgetCommand

### LINKED ISSUE #33: Add support for 'create' command for the main page parser

### PR DESCRIPTION
**Pull request by yu-ming-chen**

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -19,8 +20,8 @@
 public class SampleDataUtil {
     public static Budget[] getSampleBudgets() {
         return new Budget[] {
-            new Budget("Test Title 1", Arrays.asList(getSampleExpenditures())),
```
Comment (sogggy) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:22]: I think this portion shouldn't be changed!

File: src/main/java/seedu/address/storage/JsonAdaptedBudget.java [hunk 2]
```diff
@@ -30,7 +31,7 @@ public JsonAdaptedBudget(@JsonProperty("title") String title,
      * Converts a given {@code Budget} into this class for Jackson use.
      */
     public JsonAdaptedBudget(Budget source) {
-        title = source.getTitle();
+        title = source.getName();
```
Comment (sogggy) [inline src/main/java/seedu/address/storage/JsonAdaptedBudget.java]: change the variable name to 'name'

File: src/main/java/seedu/address/storage/JsonAdaptedBudget.java [hunk 3]
```diff
@@ -42,7 +43,7 @@ public JsonAdaptedBudget(Budget source) {
      * @throws IllegalValueException if there were any data constraints violated.
      */
     public Budget toModelType() throws IllegalValueException {
-        Budget budget = new Budget(title, new ArrayList<Expenditure>());
+        Budget budget = new Budget(new BudgetName(title), new ArrayList<Expenditure>());
```
Comment (sogggy) [inline src/main/java/seedu/address/storage/JsonAdaptedBudget.java]: change variable name to name

**Review by sogggy — CHANGES_REQUESTED** [review]
Just a few nits before approving

**Review by sogggy — DISMISSED** [review]
LGTM!

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (yu-ming-chen) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:22]: Alright will update

**Review by wenhaogoh — APPROVED** [review]
LGTM

**Review by sogggy — APPROVED** [review]
LGTM

**Pull request merged**